pkg/scheduler/apis/config/types.go

```diff
@@ -86,6 +87,8 @@ type KubeSchedulerConfiguration struct {
 	// Value must be non-negative integer. The value zero indicates no waiting.
 	// If this value is nil, the default value will be used.
 	BindTimeoutSeconds *int64
+	Plugins            Plugins
```

Comment:
These will need docstrings. Feel free to reach out to me if you're not sure what to write.